components/org.wso2.carbon.identity.api.server.idv.provider/org.wso2.carbon.identity.api.server.idv.provider.v1/src/main/java/org/wso2/carbon/identity/api/server/idv/provider/v1/impl/IdvProvidersApiServiceImpl.java

```diff
@@ -37,8 +37,15 @@
  */
 public class IdvProvidersApiServiceImpl implements IdvProvidersApiService {
 
-    @Autowired
-    IdVProviderService idVProviderService;
+    private final IdVProviderService idVProviderService;
+
+    public IdvProvidersApiServiceImpl() {
+        try {
```

Comment:
```suggestion

        try {
```

Reply:
will address this in a separate PR, as this needs to be fixed in all the components